answers/chapter9.md

```diff
@@ -0,0 +1,37 @@
+
+  1. `Throwable` is at the top of the exception hierarchy. The subclasses of Throwable are Exception and Error.
+  
+  2. ```java
+  try {
+  	// Code that might cause an error
+  }
+  catch( ExceptionType e ){ // The exception you specifically want to handle
+    // The action that the program should take if the exception occurs
+  }
+  ```
+  
+  3. No try block (or at least, no close curly bracket before the catch)
+  
+  4. It bubbles up and is reported by the JVM. Your program will exit immediately.
+  
+  5. The catch statement for Exception B is effectively unreachable code (as the catch statement for A would always catch B exceptions). 
+  You'd reorder these catch statements to make this work.
+  
+  6. Yes
+  
+  7. A finally statement _could_ be the last bit of code executed before your program ends (e.g a top level exception handler), however it doesn't have to be.
+  You can use finally after any try/catch blocks in your program. 
+  
+  8. Checked exceptions need to be declared in the 'throws' clause of a method. Unchecked exceptions don't.

```

Comment:
not sure about this one, the book answer is all exceptions except RuntimeExceptions and Errors